.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+  pull_request: {} # Validate all PRs
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        ghc: ['7.10', '8.0', '8.2', '8.4', '8.6', '8.8', '8.10', '9.0']
+        include:
+        - os: macOS-latest
+          ghc: 'latest'
+        - os: windows-latest
+          ghc: 'latest'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: haskell/actions/setup@v1
+      id: setup-haskell-cabal
+      with:
+        ghc-version: ${{ matrix.ghc }}
+    - name: Update cabal package database
+      run: cabal update
+    - uses: actions/cache@v2
+      name: Cache cabal stuff
+      with:
+        path: |
+          ${{ steps.setup-haskell-cabal.outputs.cabal-store }}
+          dist-newstyle
+        key: ${{ runner.os }}-${{ matrix.ghc }}
+    - name: Build & Test
+      run: |
+        cabal sdist --ignore-project --output-directory .
+        cabal get fast-tags-*.tar.gz
+        cd fast-tags-*/
+        cabal build fast-tags:exe:fast-tags fast-tags:tests --enable-tests
+        cabal test --enable-tests --test-show-details=direct fast-tags:tests
+    - name: Haddock
+      run: |
+        cd fast-tags-*/
+        cabal haddock all
+    # Disabled, see comment in .cabal.
```

Comment:
I didn't see this comment in the cabal diff

Reply:
The comment seems to be at https://github.com/elaforge/fast-tags/blob/master/fast-tags.cabal#L111.